### PR TITLE
Add CIS compliance to the web site

### PR DESCRIPTION
**Pull request by bwhaley**
This PR adds CIS compliance to the web site. The following updates are included

* The front page has a new "Achieve compliance" section
* Compliance-related copy on the /achieve-compliance page
* A new box on the AWS pricing page encouraging readers to contact sales for compliance pricing 

A few things are outstanding before this can be merged.

1) Need the certification approved by CIS (this is currently pending a response from CIS support)
2) Hoping to have a new grunty icon for the front page
3) Need to complete and merge the CIS production deployment guide (see #225)

### CODE REVIEW

File: _data/cis-compliance.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+infrastructure:
+  - title: Lorem ipsum dolor sit amet
```
Comment (brikis98) [inline _data/cis-compliance.yml]: What's this file for?

File: _data/pricing.yml [hunk 2]
```diff
@@ -44,6 +44,13 @@ addons:
       label: "4<span class='small'>,</span>950"
       value: 4950
       monthly: false
+  cis_benchmark_compliance:
+    name: CIS Benchmark Compliance
+    details: per month
+    price:
+      label: "4<span class='small'>,</span>395"
+      value: 9950
+      monthly: false
```
Comment (brikis98) [inline _data/pricing.yml]: I'm confused... `details` says "per month"... This says `monthly: false`... The label is 4,395... The `value` is 9950... And the HTML (in another file below) says "Contact Us"?

File: pages/checkout/_checkout.html [hunk 3]
```diff
@@ -33,6 +33,7 @@ <h5 class="margin-top-none text-center no-anchor">
               <li class="check-list-disabled" id="subscription-addon-1"><span class="help-text light" title="Our paid support program where you can contact us via email or private Slack. 2 business days SLA." data-toggle="tooltip">Professional Support</span></li>
               <li class="check-list-disabled" id="enterprise-support"><span class="help-text light" title="Our Enterprise Support includes support via email, private Slack, and phone video, all with a same-business-day response time commitment." data-toggle="tooltip">Enterprise Support</span></li>
               <li class="check-list-disabled" id="subscription-addon-2"><span class="help-text light" title="An opinionated, best-practices, end-to-end tech stack that we customize to your needs, deploy into your AWS accounts, and give you 100% of the code—all in about one day." data-toggle="tooltip">Reference Architecture</span></li>
+              <li class="check-list-disabled" id="subscription-addon-3"><span class="help-text light" title="Learn how to achieve and maintain compliance with each of the CIS AWS Foundations Benchmark recommendations" data-toggle="tooltip">CIS Compliance</span></li>
```
Comment (brikis98) [inline pages/checkout/_checkout.html]: "Learn how" sounds like we provide a guide and the user has to do lots of stuff... Should we rephrase to make it clear we provide code/modules that are already compliant with those benchmarks?

File: pages/compliance/_sub-hero.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
+<div class="row row-page-header">
+  <div class="col-xs-12">
+    <h2>Compliance in hours, not months</h2>
```
Comment (brikis98) [inline pages/compliance/_sub-hero.html]: Nice 👍 

Though perhaps it should say "days" instead of "hours", as there are typically process changes you have to implement to be compliant, which takes more than hours, right?

File: pages/compliance/_sub-hero.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    <p>
+      The <a href="https://www.cisecurity.org/benchmark/amazon_web_services/">CIS AWS Foundations Benchmark</a> is an
+      objective, of consensus-driven guideline for establishing secure infrastructure on AWS. Gruntwork's
```
Comment (brikis98) [inline pages/compliance/_sub-hero.html]:
```suggestion
      objective, consensus-driven guideline for establishing secure infrastructure on AWS. Gruntwork's
```

File: pages/compliance/_sub-hero.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+      production-grade, battle-tested infrastructure as code modules are built with compliance in mind. You can leverage
```
Comment (brikis98) [inline pages/compliance/_sub-hero.html]: "Compliance in mind" seems weak: e.g., it could be interpreted as, "we always keep it in mind but make no promises to actually meet the standards." Perhaps state more clearly that Gruntwork provides production-grade modules that are already pre-configured to meet the compliance benchmarks?

File: pages/index/_learn-more.html [hunk 7]
```diff
@@ -38,3 +38,14 @@ <h2>Get expert support and training.</h2>
...
   </div>
 </div>
+
+<div class="row row-cis-compliance">
+  <div class="col-xs-10 col-xs-offset-1 col-sm-5 col-sm-push-7 col-sm-offset-0 col-md-6 col-md-push-6 col-diagram">
+    <img src="/assets/img/infrastructure-cube-icons/infrastructure-design-your-stack-grunty.png" />
+  </div>
+  <div class="col-xs-12 col-sm-7 col-sm-pull-5 col-md-6 col-md-pull-6 col-description">
+    <h2>Achieve compliance.</h2>
+    <p>Achieve compliance with the CIS AWS Foundations Benchmark. Leverage our <a href="/guides">compliance guide</a> to  deploy compliant infrastructure fully backed by code. We have partnered with the Center for Internet Security to create certified infrastructure as code modules that achieve compliance.
```
Comment (brikis98) [inline pages/index/_learn-more.html]: This makes it sound like we only have a *guide*. We should make it clear that we have a bunch of code that is already configured to be compliant out of the box!

File: _data/achieve-complaince.yml [hunk 8]
```diff
@@ -0,0 +1,136 @@
+infrastructure:
```
Comment (brikis98) [inline _data/achieve-complaince.yml]: Where is this file used? Looks like a copy/paste of the Ref Arch features?

File: _data/cis-features.yml [hunk 9]
```diff
@@ -0,0 +1,17 @@
+- title: Compliance modules
+  description: Exclusive access to a set of CIS-compliant wrapper modules.
```
Comment (brikis98) [inline _data/cis-features.yml]:
```suggestion
  description: Exclusive access to a set of CIS-compliant infrastructure code built with Terraform, Go, Python, and Bash, including modules for AWS Config, CloudTrail, VPC Flow Logs, KMS, and much more.
```

File: _data/cis-features.yml [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+
+- title: CIS certified
+  description: The Gruntwork compliance modules are certified compliant by the Center for Internet Security.
```
Comment (brikis98) [inline _data/cis-features.yml]: Oh, that's a big deal! We should use this exact phrasing on the home page too, in the `index/_learn-more.html` below.

File: _data/cis-features.yml [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+
+- title: Fast
+  description: Achieve a compliant infrastructure in a fraction of the time it would take to do by hand.
```
Comment (brikis98) [inline _data/cis-features.yml]:
```suggestion
  description: Achieve a compliant infrastructure in a fraction of the time it would take to do it from scratch.
```

File: _data/cis-features.yml [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+
+- title: Repeatable
+  description: Us the compliance modules across multiple AWS accounts.
```
Comment (brikis98) [inline _data/cis-features.yml]:
```suggestion
  description: Use the compliance modules across multiple AWS accounts, regions, and environments.
```

File: _data/cis-features.yml [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+
+- title: Secure
+  description: Built for the security minded. <a href="https://aws.amazon.com/blogs/security/announcing-industry-best-practices-for-securing-aws-resources/"">Even AWS agrees</a>!
+
+- title: Documented
```
Comment (brikis98) [inline _data/cis-features.yml]:
```suggestion
- title: Documented and tested
```

File: _data/cis-features.yml [hunk 14]
```diff
@@ -0,0 +1,17 @@
...
+  description: Thorough docs make compliance easy.
```
Comment (brikis98) [inline _data/cis-features.yml]:
```suggestion
  description: Thorough docs and automated tests make compliance easy.
```

File: pages/index/_learn-more.html [hunk 15]
```diff
@@ -38,3 +38,14 @@ <h2>Get expert support and training.</h2>
     <p><a href="/training/" class="btn btn-primary" role="button" ga-on="click" ga-event-category="home-learn-more" ga-event-action="preview-training">Preview Our Training</a></p>
   </div>
 </div>
+
+<div class="row row-cis-compliance">
```
Comment (brikis98) [inline pages/index/_learn-more.html]: Do we have space to add the CIS logo next to the AWS, GCP, and HashiCorp logo? Would make our entire offering looking more secure & production-grade!

File: pages/compliance/_sub-hero.html (hunk 4 above)
Comment (bwhaley) [reply pages/compliance/_sub-hero.html]: Yes, I guess it's likely to take days, though our eventual target might be hours, especially if integrated in to a compliance-flavored ref arch.

File: pages/checkout/_checkout.html (hunk 3 above)
Comment (bwhaley) [reply pages/checkout/_checkout.html]: Rephrased to:
> Use infrastructure as code to achieve and maintain compliance with each of the CIS AWS Foundations Benchmark recommendations

File: _data/pricing.yml (hunk 2 above)
Comment (bwhaley) [reply _data/pricing.yml]: My mistake, a vestige of an earlier revision. I've simply removed mention of CIS pricing here instead preferring "Contact Us".

File: _data/cis-compliance.yml (hunk 1 above)
Comment (bwhaley) [reply _data/cis-compliance.yml]: A vestige - will remove.

File: pages/compliance/_sub-hero.html (hunk 6 above)
Comment (bwhaley) [reply pages/compliance/_sub-hero.html]: Rephrased to:
> Gruntwork's production-grade, battle-tested infrastructure as code modules are built for compliance. Leverage them to achieve compliance with the Benchmark quickly and repeatably, avoiding the burden of a complex, drawn-out compliance project.

File: pages/index/_learn-more.html (hunk 15 above)
Comment (bwhaley) [reply pages/index/_learn-more.html]: Good idea - added the logo alongside the others. Will post a screenshot below.

File: pages/index/_learn-more.html (hunk 7 above)
Comment (bwhaley) [reply pages/index/_learn-more.html]: Makes sense. Rephrased as:
> Achieve compliance with the CIS AWS Foundations Benchmark. The Gruntwork compliance infrastructure as code modules are certified compliant by the Center for Internet Security. Leverage our <a href="/guides">compliance guide</a> for a complete walkthrough.

File: _data/achieve-complaince.yml (hunk 8 above)
Comment (bwhaley) [reply _data/achieve-complaince.yml]: Yup, unused now.

**Comment by bwhaley** [comment]
Thanks for the review! I've incorporated all that feedback.

See some screenshots below. I thought that it might be easiest to pull the branch and spin up the page locally, but perhaps this is easiest.

CIS added to the cloud logos on the front page
CIS added to the cloud logos on the front page

Achieve compliance description on the front page
Achieve compliance description on the front page. This is where I'd picture a "security grunty" icon.

The top of the achieve compliance page
The top of the achieve compliance page

Description of how compliance works.
Description of how compliance works.

Included compliance features

Updates to the AWS pricing page
Updates to the AWS pricing page

**Comment by brikis98** [comment]
>  I thought that it might be easiest to pull the branch and spin up the page locally, but perhaps this is easiest.

We should have an automatic preview environment on PRs... We built that for the new website stuff in the `mission-control` repo, but not for the static website in this repo. When the full website lives in `mission-control`, we'll have it for everything. In the meantime, screenshots make it easier to get a sense of what's going on for those of us too lazy to checkout & run code, so thank you 😁 

> CIS added to the cloud logos on the front page

That looks good, but the blue background on the shield on the left side of the logo doesn't quite fit the style of all the other logos. Do you have a totally white version of the logo? Or could you use an image editor to make it? If not, please ask Sam!

File: pages/index/_learn-more.html [hunk 16]
```diff
@@ -38,3 +38,16 @@ <h2>Get expert support and training.</h2>
...
+  <div class="col-xs-10 col-xs-offset-1 col-sm-5 col-sm-push-7 col-sm-offset-0 col-md-6 col-md-push-6 col-diagram">
+    <img src="/assets/img/infrastructure-cube-icons/infrastructure-design-your-stack-grunty.png" />
+  </div>
+  <div class="col-xs-12 col-sm-7 col-sm-pull-5 col-md-6 col-md-pull-6 col-description">
+    <h2>Achieve compliance.</h2>
+    <p>Achieve compliance with the CIS AWS Foundations Benchmark. The Gruntwork compliance infrastructure as code
+    modules are certified compliant by the Center for Internet Security. Leverage our <a href="/guides">compliance
+    guide</a> for a complete walkthrough.
```
Comment (brikis98) [inline pages/index/_learn-more.html]: A slight rephrase to consider:

Gruntwork's modules are certified compliant by the Center for Internet Security (CIS), so you can get compliance out of the box, either by having Gruntwork deploy an end-to-end CIS-compliant Reference Architecture for you, or by assembling your own architecture using our modules and compliance guide.

**Comment by brikis98** [comment]
@bwhaley How close is this and the other CIS stuff to merging? Would love to have it in the newsletter for this month, but not sure if I should wait?

File: assets/css/pages/checkout.less [hunk 17]
```diff
@@ -55,7 +55,8 @@
         font-size: 14px;
       }
       .addon-type {
-        font-size: 30px;
+        font-size: 35px;
+        font-weight: bold;
```
[the remark this answers is not in the document]
Comment (bwhaley) [reply assets/css/pages/checkout.less]: Credit goes to @hngerebara for that!

File: assets/js/checkout.js [hunk 18]
```diff
@@ -34,24 +35,45 @@ $(function () {
...
+    case 'setup-deployment':
+      _updateCheckout({
+        setup_deployment: true
+      });
+      break;
+    case 'cis-benchmark-compliance':
+      _updateCheckout({
+        cis_benchmark_compliance: true
+      });
+      break;
```
[the remark this answers is not in the document]
Comment (bwhaley) [reply assets/js/checkout.js]: Actually they can't add it to the subscription - there's only a button to contact sales, not to add it to the subscription. This code is ineffectual.

File: pages/compliance/_sub-hero.html [hunk 19]
```diff
@@ -0,0 +1,38 @@
...
+      <strong>Get a Detailed Walkthrough for compliance with the AWS Foundations Benchmark</strong><br>
+      See our guide:
+      <a href="http://gruntwork.io/guides/compliance/how-to-achieve-cis-benchmark-compliance/">How to achieve compliance with the CIS AWS Foundations Benchmark</a>.
+    </p>
+    <div class="text-center ref-arch" style="margin-top: 30px">
+      <a href="/assets/img/cis-compliance/cis-foundations-benchmark-architecture.png" title="Click to see the full size image" target="_blank">
+        <img src="/assets/img/cis-compliance/cis-foundations-benchmark-architecture.png" alt="CIS Compliance">
+      </a>
+      <small class="text-muted">An AWS account configured for compliance with the AWS CIS Foundations Benchmark.</small>
+    </div>
```
[the remark this answers is not in the document]
Comment (bwhaley) [reply pages/compliance/_sub-hero.html]: Good idea!

**Comment by bwhaley** [comment]
@brikis98 It's basically ready to ship. We're waiting on CIS to give us the official certification, and I do not know how long it will take. They don't seem very responsive. However, @josh-padnick points out that we have already paid them and we have confidence that certification will be forthcoming, so perhaps we don't need to treat their sign off as a blocker. Other than that, the only remaining item is for me to write the blog post. 

tl;dr I think we can probably announce this in the November newsletter. When did you want to publish?

**Comment by brikis98** [comment]
Roger. Ideally, we'd publish this week. I'll review that blog post now, thanks!
